src/transactions/ManageOfferOpFrame.cpp

```diff
@@ -470,4 +470,36 @@ ManageOfferOpFrame::buildOffer(AccountID const& account,
     o.flags = flags;
     return o;
 }
+
+std::unordered_set<LedgerKey>
+ManageOfferOpFrame::getLedgerKeysToPrefetch()
+{
+    // TODO (mlo) will need to deal with best offers as well
```

Comment:
do not keep TODO in code, open an issue instead

Reply:
Done
https://github.com/stellar/stellar-core/issues/2031